pkg/utils/utils.go

```diff
@@ -75,8 +74,11 @@ func Tar(root string) ([]byte, error) {
 			return err
 		}
 
-		// Make untar destination relative, instead of absolute
-		hdr.Name = strings.TrimPrefix(strings.Replace(path, root, "", -1), string(filepath.Separator))
+		// Make untar destination relative
```

Comment:
It'd be helpful to explain why we do this (i.e. to preserve the structure of the directory inside the tar)